src/ConfigProvider.php

```diff
@@ -56,11 +59,12 @@ public function getDependencies() : array
                 PidManager::class                     => PidManagerFactory::class,
                 SwooleRequestHandlerRunner::class     => SwooleRequestHandlerRunnerFactory::class,
                 ServerRequestInterface::class         => ServerRequestSwooleFactory::class,
-                StaticResourceHandlerInterface::class => StaticResourceHandlerFactory::class,
+                StaticResourceHandler::class          => StaticResourceHandlerFactory::class,
```

Comment:
Why this change? What does it accomplish, exactly?

Reply:
TBH, Nothing much - it's just  personal habit that crept in - If you want to target that particular class rather than the interface, which might be swapped out for something else, it's more explicit, rather than, fe, attaching a delegator to the interface and having to test if it's an instance of something. That's why, but it's pretty pointless I'll agree!